sonic_y_cable/y_cable.py

```diff
@@ -0,0 +1,155 @@
+#!/usr/bin/env python
+#
+# y_cable.py
+#
+#  class for implementing Y cable access and configurations
+#   API's for Y cable functionality in SONiC
+
+try:
+    import binascii
+    import sys
+
+    import sonic_platform.platform
+    from sonic_py_common import logger
+
+except ImportError as e:
+    raise ImportError(str(e) + " - required module not found")
+
+# Global logger instance for helper functions and classes
+helper_logger = logger.Logger(SYSLOG_IDENTIFIER)
+
+# Global platform_chassis instance to call get_sfp required for read/write eeprom
+platform_chassis = sonic_platform.platform.Platform().get_chassis()
+
+# definitions of the offset with width accomodated for values
+# of MUX register specs of upper page 0x04 starting at 640
+# info eeprom
+Y_CABLE_IDENTFIER_LOWER_PAGE = 0
+Y_CABLE_IDENTFIER_UPPER_PAGE = 128
+Y_CABLE_DETERMINE_CABLE_READ_SIDE = 640
+Y_CABLE_CHECK_LINK_ACTIVE = 641
+Y_CABLE_SWITCH_MUX_DIRECTION = 642
+Y_CABLE_ACTIVE_TOR_INDICATOR = 645
+Y_CABLE_ACTIVE_TOR_INDICATOR = 0
+
+
+def toggle_mux_to_torA(physical_port):
```

Comment:
Can you please add docstrings to these functions to explain what they do (e.g., explain what "torA", "torB", "active side", read side", etc. mean?

Reply:
fixed, added doc strings